src/explore-education-statistics-admin/src/pages/release/data/utils/getUnmappedLocationErrors.ts

```diff
@@ -1,23 +1,21 @@
-import { UnmappedAndManuallyMappedLocation } from '@admin/pages/release/data/utils/getApiDataSetLocationMappings';
+import { MappableLocation } from '@admin/pages/release/data/utils/getApiDataSetLocationMappings';
 import { ErrorSummaryMessage } from '@common/components/ErrorSummary';
 import { Dictionary } from '@common/types';
 import locationLevelsMap from '@common/utils/locationLevelsMap';
 import camelCase from 'lodash/camelCase';
 
 export default function getUnmappedLocationErrors(
-  unmappedAndManuallyMappedLocations: Dictionary<
-    UnmappedAndManuallyMappedLocation[]
-  >,
+  mappableLocations: Dictionary<MappableLocation[]>,
 ): ErrorSummaryMessage[] {
   const errors: ErrorSummaryMessage[] = [];
 
-  Object.keys(unmappedAndManuallyMappedLocations).forEach(level => {
-    const total = unmappedAndManuallyMappedLocations[level].filter(
+  Object.keys(mappableLocations).forEach(level => {
+    const total = mappableLocations[level].filter(
       map => map.mapping.type === 'AutoNone',
     ).length;
     if (total) {
       errors.push({
-        id: `unmapped-${level}`,
+        id: `mappable-${level}`,
         message: `There ${total > 1 ? 'are' : 'is'} ${total} unmapped ${
           total > 1
             ? // TODO remove camelCase when levels are camelCase in BE
```

Comment:
As mentioned elsewhere, we can remove the camelCasing as we're doing this in `getApiDataSetLocationMappings`.